python/pfs/drp/stella/calibrateWavelengthsTask.py

```diff
@@ -315,16 +355,24 @@ def fitWavelengthSolution(self, spec, detectorMap, rng=np.random):
             else:
                 dy = -int(-dy)
                 spec.wavelength[dy:] = spec.wavelength[:-dy]
-
+                
+        detectorMap.setWavelength(fiberId, rows, spec.wavelength)
         diff = detectorMap.getWavelength(fiberId) - spec.wavelength
         self.log.info("Fiber %d: wavelength correction %f +/- %f nm" % (fiberId, diff.mean(), diff.std()))
-        detectorMap.setWavelength(fiberId, rows, spec.wavelength)
+        
+        lines = []
+        for i, rl in enumerate(refLines):
+            if (rl.status & drpStella.ReferenceLine.Status.FIT) == 0:
+                continue    
+            fitXPosition = detectorMap.findPoint(fiberId, fitWavelength[i])[0]
+            nominalXPosition = detectorMap.findPoint(fiberId, wavelength[i])[0]
+            line=LineData(fiberId, wavelength[i], nominalXPosition, fitXPosition, nominalPixelPos[i], rl.fitPosition, fitWavelength[i], rl.fitPositionErr, fitWavelengthErr[i], wavelengthCorr(nominalPixelPos[i]),flag[i])
+            lines.append(line)
+        return [lines, wavelengthCorr]
```

Comment:
If you have to return more than one thing from a method, please use `lsst.pipe.base.Struct`, so the elements are named.